src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
+fn foo() {}
+
+#[target_feature(enable = "sse2")]
+fn bar() {
+    foo();
+}
+
+#[target_feature(enable = "avx")]
+fn baz() {
+    foo(); //~ ERROR call to function with #[target_feature] is unsafe
+}
+
+fn main() {
+    unsafe {
```

Comment:
Let's also test:
> Calling them from other contexts (other functions, static variable initializers, etc.) requires opening an unsafe {} even though they are not marked as unsafe

in particular with respect to various const contexts (e.g. static and const initializers).